### PR TITLE
Created the star block

### PR DESCRIPTION
**Pull request by olya-yevsieienko**
- [DEMO LINK](https://olya-yevsieienko.github.io/layout_stars/)
- [TEST REPORT LINK](https://olya-yevsieienko.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
```
Comment (jstmpelowycz) [inline src/style.css:10]: You don't need flexbox here, as long as you have nothing to align inside the `stars__star` div :)

**Review by jstmpelowycz — APPROVED** [review]
Good job!

**Comment by olya-yevsieienko** [comment]
Okay, I understood. Thanks!

вт, 10 мая 2022 г. в 13:47, Dmytro Pelovych ***@***.***>:

**Review by jstmpelowycz — APPROVED** [review]
Re-approve

**Pull request closed without merging**